modules/nosql/manager_redis_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
```

Comment:
sorry it's actually 22 - I'm somethimes living in the past ;)

Reply:
All good! The future is here!